files/image_config/warmboot-finalizer/finalize-warmboot.sh

```diff
@@ -152,4 +169,7 @@ if [[ -n "${list}" ]]; then
     debug "Some components didn't finish reconcile: ${list} ..."
 fi
 
+if [ ${FAST_REBOOT} == "true" ]; then
+    finalize_fast_reboot
+fi
 finalize_warm_boot
```

Comment:
`finalize_warm_boot` is being called regardless of system undergoing fast/warm boot. Is that needed? We can put that into conditional block for warm reboot.